contracts/structs/JBConfigurationFundingCycleMetadataData.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+struct JBConfigurationFundingCycleMetadataData {
+  // The configuration during which the new metadata was set.
+  uint56 configuration;
```

Comment:
Curious about the actual cost diff of storing as uint256 (since it's not packable anyways) and avoiding downsizing/typecasting costs vs. storing as a smaller uint56